docs/markdown/Builtin-options.md

```diff
@@ -281,3 +282,14 @@ relative to the installation `prefix`, which will often result in installed pyth
 modules to not be found by the interpreter unless `prefix` is `/usr` on Linux,
 or for example `C:\Python39` on Windows. These options can be absolute paths
 outside of `prefix`.
+
+*Since 0.61.0* The `python.install_env` option is used to detect the correct
```

Comment:
It's will have to wait for 0.62.0 sadly, right?

Reply:
Yeah, just never rebased it yet.